llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -469,14 +464,14 @@ class InstrProfSymtab {
   /// until before it is used. See also \c create(StringRef) method.
   Error create(object::SectionRef &Section);
 
-  /// This interface is used by reader of CoverageMapping test
-  /// format.
-  inline Error create(StringRef D, uint64_t BaseAddr);
-
   /// \c NameStrings is a string composed of one of more sub-strings
   ///  encoded in the format described in \c collectPGOFuncNameStrings.
   /// This method is a wrapper to \c readPGOFuncNameStrings method.
-  inline Error create(StringRef NameStrings);
+  Error create(StringRef NameStrings);
+
+  /// This interface is used by reader of CoverageMapping test
+  /// format.
+  inline Error create(StringRef D, uint64_t BaseAddr);
```

Comment:
Is this function missing?

Reply:
I think this is an artifact of 'diff' UI.

I swapped the order of two functions(https://github.com/llvm/llvm-project/blob/fb8ff4cdaecb7fe498d4e3a9cbcb33d40cccd29a/llvm/include/llvm/ProfileData/InstrProf.h#L472-L479) primarily because, the second `create` no longer has an 'inline' keyword (since its definition is moved to cpp files).

Reply:
`ninja check-llvm` pass (without compiler errors).